models/UserDao.js

```diff
@@ -1,5 +1,20 @@
 import prisma from '../prisma';
 
+const createUsers = async (name, email, userAccount, phoneNumber, hashedPw) => {
+  return await prisma.$queryRaw(`
+    INSERT INTO users(
+      name, 
+      email, 
+      user_account, 
+      phone_number, 
+      password)
+    SELECT '${name}', '${email}', '${userAccount}', '${phoneNumber}', '${hashedPw}'
+    WHERE NOT EXISTS 
+    (SELECT user_account FROM users 
+      WHERE user_account = '${userAccount}')
+  `);
+};
+
```

Comment:
```suggestion
  return await prisma.$queryRaw(`
    INSERT INTO users(
      name, 
      email, 
      user_account, 
      phone_number, 
      password)
    VALUES( '${name}', '${email}', '${userAccount}', '${phoneNumber}', '${hashedPw}')
    WHERE NOT EXISTS 
    (SELECT user_account FROM users 
      WHERE user_account = '${userAccount}')
  `);
};

```
이렇게 넣으시는 것이 좋습니다 :)

`INSERT INTO SELECT`는 다른 테이블에 있는 값을 복사해서 넣을 때 사용하는 구문입니담!
https://www.w3schools.com/sql/sql_insert_into_select.asp